app/controllers/user.controller.js

```diff
@@ -244,3 +247,146 @@ exports.deleteAll = (req, res) => {
       });
     });
 };
+
+
+// Reset password
+
+exports.resetPassword = async (req, res) => {
```

Comment:
wasn't this function already merged as part of another PR?  why is it showing up here in this PR?

Reply:
it's because you aren't starting new branches from the `dev` branch, but from a previous feature branch:
![image](https://github.com/burra162/stories-app-backend/assets/19157011/b86c8752-0303-4574-91cb-4012cfce2af4)

if you need help understanding how to properly start a branch from `dev`, please ask me